.ebextensions/deploy.config

```diff
@@ -0,0 +1,118 @@
+option_settings:
+  - namespace: aws:autoscaling:launchconfiguration
+    option_name: BlockDeviceMappings
+    value: /dev/sdh=:10
+  - option_name: BUNDLE_DISABLE_SHARED_GEMS
+    value: "1"
+  - option_name: BUNDLE_PATH
+    value: "vendor/bundle"
+
+packages:
+  yum:
+    git: []
+
+files:
+  "/etc/nginx/conf.d/proxy.conf" :
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      client_max_body_size  200M;
+      proxy_connect_timeout 1800;
+      proxy_send_timeout    1800;
+      proxy_read_timeout    1800;
+      send_timeout          1800;
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/03_webapp_user.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      #!/usr/bin/env bash
+      sudo chown -R webapp:webapp /home/webapp
+      sudo service nginx reload
+      true
+  "/opt/elasticbeanstalk/hooks/appdeploy/pre/13_ebs_perm.sh":
+    mode: "000755"
+    owner: root
+    group: root
+    content: |
+      #!/usr/bin/env bash
+      if ! mount | grep /dev/xvdh > /dev/null; then
+        sudo mkfs -t ext3 /dev/sdh
+        sudo mkdir /media/volume1
+        sudo mount /dev/sdh /media/volume1
+      fi
+      sudo groupadd appdev
+      sudo usermod -aG appdev ec2-user
+      sudo usermod -aG appdev webapp
+      sudo chown -R webapp:appdev /media/volume1
```

Comment:
I think this line (48) is unnecessary because the same line appears below, and none of the lines between these two identical chown lines require the ownership change.